scripts/dartfmt.sh

```diff
@@ -24,6 +26,18 @@ $DARTFMT -l 62 examples/language_tour/flow/exceptions/web/main.dart
 
 $DARTFMT -l 75 examples/language_tour/flow/break-continue/web/main.dart
 
+# New code
+
+$DARTFMT -l 60 \
+  examples/lib/language_tour/classes/immutable_point.dart \
+  examples/lib/language_tour/classes/logger_with_main.dart \
+  examples/lib/language_tour/classes/no_such_method.dart \
+  examples/lib/language_tour/classes/orchestra.dart \
+  examples/test/language_tour/built_in_types_test.dart \
+  examples/test/language_tour/functions_test.dart
+
+$DARTFMT -l 65 examples/lib/language_tour/exceptions.dart
```

Comment:
Why the different line lengths for different examples?

(We should figure out what the optimum line length is, testing with various screen widths. But that can wait.)

Reply:
In the previous pass over the code, I determined the line lengths by experimentation. My goal was to ensure that we could re-run dartfmt and preserving the existing layout of the code excerpts.

Reply:
Should we just make the first `$DARTFMT` call (line 13) have `-l 60`? Or do more samples require the default (80)?

Reply:
IIRC (which I might not), the maximum code line length for most pages we checked was 68. I'd be happy with a default 65- or 60-character line length.

Reply:
Most samples are fine when the default 80 is used.

Reply:
I'd like to avoid horizontal scrolling, wherever possible. 80 characters used to be too long for the window width I usually use.

Reply:
> I'd be happy with a default 65- or 60-character line length.

Ok, I can try 60 for all samples and show you the result.

Reply:
See #431, and let me know what you think.